src/lib/events/event.js

```diff
@@ -0,0 +1,66 @@
+import EventData from '@/lib/events/event-data.js'
+
+/**
+ * An event in pub/sub (publish–subscribe) design pattern
+ */
+export default class Event {
```

Comment:
@kirlat, may be I am wrong here - but you are overwritten a global Event object on your own without any inheritance.
Does it mean that there is no more standard [Event](https://developer.mozilla.org/en-US/docs/Web/API/Event/Event) and all its method are no more applicable?

Reply:
I agree that it's better not to overwrite a global event object. I will rename the `Event`.